cloud/edgecontroller/pkg/devicecontroller/config/load.go

```diff
@@ -0,0 +1,19 @@
+package config
+
+import (
+	"github.com/kubeedge/beehive/pkg/common/config"
+	"github.com/kubeedge/beehive/pkg/common/log"
+	"github.com/kubeedge/kubeedge/cloud/edgecontroller/pkg/devicecontroller/constants"
+)
+
+// UpdatePodStatusWorkers is the count of goroutines of update pod status
+var UpdateDeviceStatusWorkers int
+
+func init() {
+	if psw, err := config.CONFIG.GetValue("devicecontroller.update-pod-status-workers").ToInt(); err != nil {
```

Comment:
Does device controller should take care of pod status? And why we name this file `load.go`?

Reply:
It was a typo, It is deviceStatus. Thanks.
It is named load.go as it is used for loadbalancing by starting multiple workers if required.